nailgun/entity_mixins.py

```diff
@@ -979,6 +979,9 @@ def update_payload(self, fields=None):
 
         """
         values = self.get_values()
+        updatable_fields = (self.updatable_fields if hasattr(self, 'updatable_fields') else
+                            values.keys())
+        values = {field: value for field, value in values.items() if field in updatable_fields}
```

Comment:
I don't think the current unit test covers this behavior, can you please update the current unit test class for this mixin to include an update on a class with `updatable_fields` defined?